src/Cassandra/Metrics/StubImpl/EmptyDriverTimer.cs

```diff
@@ -0,0 +1,14 @@
+using Cassandra.Metrics.DriverAbstractions;
+
+namespace Cassandra.Metrics.StubImpl
```

Comment:
NIT: I would prefer to avoid names like `Stub` because people usually associate `stubs`, `fakes`, `mocks` etc to test code when it's not the case here. I would prefer something like `Null`, `Empty` or `Noop` (this last one is stolen from the java driver 😄 )

Reply:
Fixed